install-sbo.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -e
+set -e
+set -u
+REGISTRY_PASSWORD?=""
+
+#The SBO can also be installed in a vanilla kubernetes cluster. A prerequisite for this would be to add credentials for the registry.redhat.io in the cluster. Steps to be followed to achieve the same are listed below:
+#1) Follow [Red Hat Container Registry Authentication Steps](https://access.redhat.com/RegistryAuthentication)
+#2) Verify that your credentials are correct using docker login -u <your_username> -p <your_passwd> registry.redhat.io
+echo "${REGISTRY_PASSWORD}" | docker login -u "$USERNAME" --password-stdin registry.redhat.io
+
+#3) Start a kubernetes cluster. For this case we have used minikube. Any other Kubernetes cluster can be used as well.
+minikube start
+
+#4) Enable [OLM](https://github.com/operator-framework/operator-lifecycle-manager) in the cluster by running the following command
+curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/0.16.1/install.sh | bash -s 0.16.1
```

Comment:
we should check if OLM is enabled and if not install it. Extractg OLM version into some variable to make the maintenance easier.

Reply:
If OLM is already installed it would just show as a resource `unchanged` in case of an `apply` API request